src/analysis/retail/monk/mistweaver/modules/spells/JadeSerpentStatue.tsx

```diff
@@ -147,15 +148,9 @@ class JadeSerpentStatue extends Analyzer {
         size="flexible"
         category={STATISTIC_CATEGORY.TALENTS}
       >
-        <BoringValueText
-          label={
-            <>
-              <SpellIcon id={TALENTS_MONK.SUMMON_JADE_SERPENT_STATUE_TALENT.id} /> % Uptime
-            </>
-          }
-        >
-          <>{formatPercentage(this.jadeSerpentStatueUptime)}</>
-        </BoringValueText>
+        <TalentSpellText talent={TALENTS_MONK.SUMMON_JADE_SERPENT_STATUE_TALENT}>
+        <>{formatPercentage(this.jadeSerpentStatueUptime)}% Uptime</>
```

Comment:
You don't need to wrap this in a fragment, it's already being wrapped by the surrounding outside component.